.travis.yml

```diff
@@ -9,8 +9,11 @@ go:
   - 1.5
 
 before_install:
-  - sudo apt-get update
-  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
+  - echo "Dpkg::Options { \"--force-confdef\"; \"--force-confold\"; }" |sudo tee -a /etc/apt/apt.conf.d/local
+  - sudo apt-get install apt-transport-https ca-certificates
+  - curl -sSL https://get.docker.com/ | sudo sh
+  - sudo pip2 install --upgrade pip setuptools wheel docker-compose
```

Comment:
The integration tests don't need `docker-compose`. We maybe could add some more tests to test the "run as a Docker container" version and use `docker-compose`, but for now, this isn't needed.